templates/common/README.md

```diff
@@ -0,0 +1,6 @@
+## Scaffold Circuit
```

Comment:
First use of the common template.  Definitely could have supplied more details here, but I think it makes sense to wait and see how the other framework scaffolds shape up. I think it makes sense for all the scaffolds to have a x!=y circuit